data/json/items/resources/plastic.json

```
@@ -54,5 +54,18 @@
     "symbol": ")",
     "color": "light_blue",
     "use_action": { "type": "deploy_furn", "furn_type": "f_plastic_groundsheet" }
+  },
+  {
+    "id": "rigid_plastic_sheet"
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": "plastic sheet",
+    "description": "This is a large sheet of heavy plastic, the sort that might have been used for windows.",
+    "weight": "1000 g",
+    "volume": "2 L",
```

```suggestion
    "weight": "14875 g",
    "volume": "12500 ml",
```
I used information from here: (https://www.acplasticsinc.com/informationcenter/r/guide-to-plastic-weights ), specifically acrylic, and based volume on glass panes. Turns out glass is seriously light in game, I'll do a PR for that. It's currently pretty close to the weight I got for acrylic, which is what I went for as a medium. I'm _pretty sure_ acrylic is common. Here's the calculator: (https://www.calculator.net/density-calculator.html?cdensity=1.19&cdensityunit=gram%2Fcubic+centimeter&cvolume=12.5&cvolumeunit=liter&cmass=&cmassunit=kilogram&ctype=&x=40&y=38)

You should add this to some loot tables, too, or it won't spawn.

---

I'm going to go over these, I not sure how to spawn thing just yet I'm going to look into that shortly or just look at someone else's mod